src/sass/app_2/_common/base/themes.scss

```diff
@@ -87,5 +89,8 @@ $themes: (
         tooltip_background_color         : $COLOR_DARK_BLUE_5,
         tooltip_background_error         : $COLOR_RED,
         won_gradient_color               : $COLOR_DARK_GREEN_GRADIENT,
+        wip_btn_color                    : $COLOR_DARK_BLUE_2,
+        wip_color                        : $COLOR_WHITE,
+        loader_background                : $COLOR_WHITE,
```

Comment:
we can move loader background to make it alphabetical

Reply:
actually loader_background is duplicated, I'll remove it